presentation/src/main/res/layout/fragment_bottom_sheet_faq.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/faqBottomSheet"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingBottom="@dimen/size_2x"
+    android:background="@drawable/rounded_layout_top_bg">
+
+    <View
+        android:id="@+id/view"
+        android:layout_width="80dp"
+        android:layout_height="3dp"
+        android:layout_marginTop="@dimen/size_1.5x"
+        android:background="@color/n_70"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/question_text_view"
+        style="@style/Text_comforta_bold_18sp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/size_4x"
+        android:layout_marginStart="@dimen/size_3x"
+        android:text="@string/lorem_ipsum_dolor_sit_amet"
+        android:textColor="@color/s_60"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/view" />
+
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/scrollView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintHeight_max="600dp"
+        app:layout_constraintBottom_toTopOf="@id/btnOk"
+        android:layout_marginBottom="@dimen/size_2x"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/question_text_view"
+        app:layout_constraintVertical_bias="0.0">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/answer_text_view"
+                style="@style/Poppins_regular_14"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/size_3x"
+                android:layout_marginTop="@dimen/size_2x"
+                android:layout_marginEnd="@dimen/size_3x"
+                android:layout_marginBottom="@dimen/size_2x"
+                android:text="@string/medium_lorem_ipsum" />
+        </LinearLayout>
+
+    </androidx.core.widget.NestedScrollView>
+
+    <Button
+        android:id="@+id/btnOk"
```

Comment:
Shadow of this button is cut a bit :
![image](https://github.com/SafetyHeads/Akademia-Androida-2023/assets/3758285/d1ebf6f5-2c19-4857-81cc-f9406560f249)